CHANGES.rst

```diff
@@ -2,10 +2,11 @@ Changelog
 =========
 
 
-2.0b1 - Unreleased
-------------------
+3.0 - Unreleased
+----------------
```

Comment:
Version 2.0 final was never released.  Are we skipping it entirely?  Why?

Reply:
You are right, 2.0 should be enough.